eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -40,7 +40,7 @@ steps:
       -BaseName 'Generated'
       -ServiceDirectory '${{ parameters.ServiceDirectory }}'
       -TenantId '$(aad-azure-sdk-test-tenant-id-gov)'
-      -SubscriptionId '$(azure-subscription-id)'
+      -SubscriptionId '$(azure-subscription-id-gov)'
```

Comment:
Did you make sure these were in the variable group? `azure-subscription-id` wasn't until I added it, though there was an `azure-subscription` variable that doesn't actually exist in our Key Vault and you may want to remove just for confusion (I didn't want to chance it, despite it seeming safe after what happened last year).

Reply:
I've added them in the provisioner variable group.